### DISCUSSION TITLE
Being able to select the FormBuilderRadioGroup options by clicking on the optinos label line

### DISCUSSION THREAD

**Discussion by Natan-hub** [danvick/flutter_form_builder#1356] [General]
Nowadays, if you create a FormBuilderRadioGroup, I can only select an option by clicking exactly on the radio as shown in the photo, highlighted by the orange color. I would like to know if it can be implemented or if it is possible to select an option by also clicking on the area highlighted in green

![image](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/67711835/815d5d18-ad0a-4001-bf86-9b6c74a62726)

**Comment by deandreamatias** [comment]
This would be nice to create issue of type feature

**Reply by deandreamatias** [reply]
This still an issue? @Natan-hub
